# HG changeset patch
# User Florian Pose
# Date 1543419697 -3600
#      Wed Nov 28 16:41:37 2018 +0100
Bumped library version.
(grafted from ffdca5faecf6710bffd8ee549f6d8ed4b7deffe3)

diff --git a/lib/Makefile.am b/lib/Makefile.am
--- a/lib/Makefile.am
+++ b/lib/Makefile.am
@@ -51,7 +51,27 @@ noinst_HEADERS = \
 	voe_handler.h
 
 libethercat_la_CFLAGS = -fno-strict-aliasing -Wall -I$(top_srcdir)
-libethercat_la_LDFLAGS = -version-info 1:0:0
+
+#
+# http://www.gnu.org/software/libtool/manual/html_node/ ...
+# Updating-version-info.html
+#
+# - If the library source code has changed at all since the last update, then
+#   increment revision (‘c:r:a’ becomes ‘c:r+1:a’).
+# - If any interfaces have been added, removed, or changed since the last
+#   update, increment current, and set revision to 0.
+# - If any interfaces have been added since the last public release, then
+#   increment age.
+# - If any interfaces have been removed or changed since the last public
+#   release, then set age to 0.
+#
+# History
+#
+# 1:0.0
+# 	ecrt_master_sync_reference_clock_to() added.
+# 2:0:1
+#
+libethercat_la_LDFLAGS = -version-info 2:0:1
 
 #------------------------------------------------------------------------------
 
